flink-rpc/flink-rpc-akka-loader/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcSystemLoader.java

```diff
@@ -42,6 +43,15 @@
  */
 public class AkkaRpcSystemLoader implements RpcSystemLoader {
 
+    /**
+     * The name of the akka dependency jar, which should be correspond to the artifact output copied
+     * to flink-runtime module.
+     */
+    private static final String FLINK_RPC_AKKA_FAT_JAR = "flink-rpc-akka.jar";
+
+    protected static final String HINT_USAGE =
```

Comment:
```suggestion
    static final String HINT_USAGE =
```
Seems more appropriate.